docs/autogenerated/openapi.yaml

```diff
@@ -0,0 +1,602 @@
+openapi: 3.0.1
+info:
+  title: ecChronos REST API
+  description: ecChronos REST API can be used to view repair status of tables and
+    run repairs
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0
+  version: 1.0.0
```

Comment:
Switch order of license and version to mirror what you did in RestManagmentRESTImpl